src/client/fuse_ll.cc

```diff
@@ -612,9 +613,30 @@ static void fuse_ll_read(fuse_req_t req, fuse_ino_t ino, size_t size, off_t off,
   Fh *fh = reinterpret_cast<Fh*>(fi->fh);
   bufferlist bl;
   int r = cfuse->client->ll_read(fh, off, size, &bl);
-  if (r >= 0)
-    fuse_reply_buf(req, bl.c_str(), bl.length());
-  else
+  if (r >= 0) {
+    vector<iovec> iov;
+    bl.prepare_iov(&iov);
+#ifdef CEPH_HAVE_SPLICE
+    size_t len = sizeof(struct fuse_bufvec) + sizeof(struct fuse_buf) * iov.size() - 1;
```

Comment:
why '-1'

Reply:
```
struct fuse_bufvec {
....
        struct fuse_buf buf[1];                                                                                                            
};
```
For the array count of `fuse_buf` in `struct fuse_bufvec` what we need is `iov.size()` and the `struct fuse_buffvec` will have extra one.

Reply:
should be sizeof(struct fuse_buf)  * (iov.size() - 1)

Reply:
Yeah, right. Will fix it.